waku/waku_archive/archive.nim

```diff
@@ -94,26 +94,37 @@ proc handleMessage*(
 
   let
     msgDigest = computeDigest(msg)
+    msgDigestHex = msgDigest.data.to0xHex()
     msgHash = computeMessageHash(pubsubTopic, msg)
+    msgHashHex = msgHash.to0xHex()
     msgTimestamp =
       if msg.timestamp > 0:
         msg.timestamp
       else:
         getNanosecondTime(getTime().toUnixFloat())
 
   trace "handling message",
+    msg_hash = msgHashHex,
     pubsubTopic = pubsubTopic,
     contentTopic = msg.contentTopic,
     msgTimestamp = msg.timestamp,
     usedTimestamp = msgTimestamp,
-    digest = toHex(msgDigest.data),
-    messageHash = toHex(msgHash)
+    digest = msgDigestHex
 
   let insertStartTime = getTime().toUnixFloat()
 
   (await self.driver.put(pubsubTopic, msg, msgDigest, msgHash, msgTimestamp)).isOkOr:
     waku_archive_errors.inc(labelValues = [insertFailure])
     debug "failed to insert message", err = error
+
+  info "message archived",
```

Comment:
One suggestion i have is rather than making all these as info logs? Can we make it as part of compile flag or some config to log such messages for debugging/tracing? I remember Jakubs mentioning something similar being already there in nwaku which they enable in status fleets for tracing.